Data/Scripts/GardenConquest/Messaging/ResponseProcessor.cs

```diff
@@ -66,14 +66,32 @@ public class ResponseProcessor {
 				// Deserialize the message
 				BaseResponse msg = BaseResponse.messageFromBytes(buffer);
 
+				// Check for missing important parts
+				if (msg.Destination == null || msg.DestType == null) {
```

Comment:
This, along with the two commented sections below, was a small remnant of the logic I put in place to help debug this initially - they just seemed useful to keep for a while until more stable. But I understand why the logic here is flawed and the others a pretty simple to reproduce, so I'll remove it all.